crates/common/validator/lean/src/service.rs

```diff
@@ -74,16 +75,20 @@ impl ValidatorService {
                                     keystore.validator_id,
                                 );
 
-                                // TODO: Sign the block with the keystore.
-                                let signed_block = SignedBlock {
-                                    message: new_block,
-                                    signature: FixedBytes::default(),
+                                // TODO: Sign the block with the keystore once spec is finalized.
+                                let signed_block = SignedBlockWithAttestation {
+                                    message: BlockWithAttestation {
+                                        block: new_block,
+                                        proposer_attestation: Attestation::default(),
+
+                                    },
+                                    signature:VariableList::empty(),
```

Comment:
```suggestion
                                    signature: VariableList::empty(),
```

how is fmt ci passing